share/isso.conf

```diff
@@ -249,3 +249,15 @@ base =
 
 # Limit the number of elements to return for each thread.
 limit = 100
+
+[webhook]
+# Isso can notify you on new comments via web hook.
+# By default, it sends a POST data with new comment metadata: author, author email, author website, text, moderation URLs (activation, deletion, view).
+# It's also possible to add a JSON template (using Python string.Template) to customize the POST data. Useful to fit some tools abilities ike Slack, Matrix, Teams, etc.
```

Comment:
"It is also possible to add a JSON template (using Python string.Template) to customize the POST request data. Useful to integrate with some tools like e.g. Slack, Matrix or MS Teams."